src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -59,6 +59,9 @@
             <argument type="service" id="twig" />
             <argument type="service" id="templating.name_parser" />
             <argument type="service" id="templating.locator" />
+
+            <autowiring-type>Symfony\Component\Templating\EngineInterface</autowiring-type>
+            <autowiring-type>Symfony\Bundle\FrameworkBundle\Templating\EngineInterface</autowiring-type>
```

Comment:
No, the last defined take precedence. If both are loaded, twig will be autowired (it it is registered after FrameworkBundle).